src/bash/sdv-health

```diff
@@ -362,34 +360,34 @@ if [ -n "$SDV_SERVICES_OPT" ]; then
 	done
 fi
 
-K3S_IS_ACTIVE=$(systemctl is-active k3s)
-if [ -n "$SDV_PODS" ]; then
+CM_STATUS=$(systemctl is-active container-management)
+if [ -n "$KANTO_CM_CONTAINERS" ]; then
 	printf -- "$SEPARATOR\n"
 
-	printf -- "${COL_WHITE}[SDV Pods]${COL_NC}\n"
-	if [ $K3S_IS_ACTIVE ]; then
+	printf -- "${COL_WHITE}[Kanto CM Containers]${COL_NC}\n"
+	if [[ ${CM_STATUS} != *"inactive"*  ]]; then
 
-		ALL_PODS=$(kubectl get pods -A -o custom-columns=:metadata.name,:status.phase --no-headers=true --request-timeout='20s' --sort-by=metadata.name)
-		for expectedPod in $SDV_PODS; do
+		# removes tabs, splits on pipe and takes the container name column and the container status colum (2 and 4)
+		ALL_CNTRS=$(${KANTO_CMD} list | awk -F'|' '{gsub(/\t/, ""); print $2 $4}') 
+		for expectedCntr in $KANTO_CM_CONTAINERS; do
 			while IFS= read -ra line ; do
 				#echo $line;
 				IFS=' ' read -ra ARR <<< $line
-				#printf '  * %-23s : %s\n' "${ARR[0]}" "${ARR[1]}"
-				pod=${ARR[0]}
+				cntr=${ARR[0]}
 				status=${ARR[1]}
 
-				if [[ "$pod" == *"$expectedPod"* ]]; then
-					if [ "$status" = "Running" ] || [ "$status" = "Succeeded" ]; then
-						printf "  * %-40s : $TEXT_OK\n" "${pod}"
+				if [[ "$cntr" == *"$expectedCntr"* ]]; then
+					if [ "$status" = "Running" ] || [ "$status" = "Exited" ]; then # <------------------- What states do we put here?
```

Comment:
We're only looking for `Running` containers (services). One-off containers are not interesting for the sdv-health script.

Hence, the status `Exited` is considered a failure of the service and thus should be displayed as an error.

Reply:
"Exited" state removed.